### PR TITLE
org.graalvm.polyglot.Context & BinaryString

### PR DESCRIPTION
**Pull request by konikvranik**
- support for ES6 #101
- preparation for proper binary files handling #70

### CODE REVIEW

**Comment by quangson91** [comment]
@konikvranik Thank you so much for creating a PR.
Did you finish this PR?
I would like to check & merge.

Thanks!

**Comment by konikvranik** [comment]
Hi @quangson91, you're free to review & merge. For next feature (#110 ) I've created extra PR. So I'll wait for you until I'll merge it into mine master (which is source for this PR)

**Comment by quangson91** [comment]
@konikvranik 
There is some unit test that failed.
Would you mind checking it?
Otherwise, I will check it later.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
           java-version: 11
 
       - name: Clean & Build Jar
-        run: ./gradlew clean test jar
```
Comment (quangson91) [inline .github/workflows/build.yml:33]: We should keep `test`.
So in the future, if there is change, we will easier to find the problem.

**Comment by konikvranik** [comment]
I've just noticed. There is no reason to fail the test. Probably just single run failure. I removed the redundant workflow I added by mistake. Now it should pass.

File: .github/workflows/build.yml (hunk 1 above)
Comment (konikvranik) [reply .github/workflows/build.yml:33]: BTW, in the gradle there is `check` task which should be wider than just the `test`. Build is including the  `check`

Comment (quangson91) [reply .github/workflows/build.yml:33]: Yup. You are right.

**Review by quangson91 — COMMENTED** [review]
Just have a small question.
Otherwise, LGTM!

File: src/main/java/uos/dev/restcli/HttpRequestFilesExecutor.kt [hunk 2]
```diff
@@ -122,6 +122,7 @@ class HttpRequestFilesExecutor constructor(
             val rawRequest = requests.getOrNull(requestIndex) ?: return
 
             runCatching {
+                jsClient = JsClient()
```
[the remark this answers is not in the document]
Comment (konikvranik) [reply src/main/java/uos/dev/restcli/HttpRequestFilesExecutor.kt:125]: That's why with Graal everything was held on context. So if you created `const` in one of requests and then you tried torepeate it with the same name in the another, it start's to complain that this already exists. So now for every request must me extra context.

**Comment by quangson91** [comment]
@konikvranik I merged. Thank you so much!

**Comment by konikvranik** [comment]
@quangson91 Thank you for the project. :-)

**Pull request merged**